content/2. Ondersteunende-informatie/3. Processen/4. Implementatieproces/2. Bouwen softwaresysteem/While/1. While.md

```diff
@@ -0,0 +1,62 @@
+---
+title: 1. While
+taxonomie:
+  - ib-19.2.Controlestructuur-Herhaling-While
+  - ib-19.3.Controlestructuur-Herhaling-While
+draft: false
+---
+
+## Wat is een While-loop?
+In de programmeerwereld worden **While-loops** gebruikt om een bepaald stukje code herhaaldelijk uit te voeren zolang aan een voorwaarde wordt voldaan. Dit maakt het mogelijk om dynamische herhalingen te creëren, waarbij je vooraf niet weet hoe vaak de loop zal draaien. De herhaling stopt zodra de voorwaarde **onwaar** (false) wordt.
+
+Zonder while-loops zou je telkens dezelfde code handmatig moeten herhalen, wat onpraktisch en foutgevoelig is. Met een while-loop kan een programma zichzelf herhalen en aanpassen op basis van variabele waarden en voorwaarden.
+
+> [!TIP] Casus
+> Stel je telt tot 5. Je begint bij 1 en blijft tellen totdat je bij 5 komt. Zodra je bij 5 bent, stop je met tellen.
+
+## Hoe zit een While-loop in elkaar?
+De while-loop volgt een specifieke opbouw die begint met een while-clausule waarin een conditie wordt gespecificeerd. Zolang deze conditie waar (`true`) is, wordt de code binnen de lus herhaaldelijk uitgevoerd. Zodra de conditie onwaar (`false`) wordt, stopt de herhaling en gaat het programma verder.
+
+Een conditie is een voorwaarde waaraan iets, zoals een variabele, kan voldoen.
+
+### Codevoorbeeld
+```C#
+while(conditie){
+	// code om uit te voeren zolang de conditie waar is
```

Comment:
Misschien dat het de webeditor is, maar de indentatie van de comment is te groot 
![image](https://github.com/user-attachments/assets/250e118e-489c-46fb-b0ba-4b8c64e0d05f)